src/TokenizedStrategy.sol

```diff
@@ -642,14 +677,8 @@ contract TokenizedStrategy {
      * @param shares The amount of the strategies shares.
      * @return . Expected amount of `asset` the shares represents.
      */
-    function convertToAssets(uint256 shares) public view returns (uint256) {
-        // Saves an extra SLOAD if totalSupply() is non-zero.
-        uint256 supply = totalSupply();
-
-        return
-            supply == 0
-                ? shares
-                : shares.mulDiv(totalAssets(), supply, Math.Rounding.Down);
+    function convertToAssets(uint256 shares) external view returns (uint256) {
+        return _convertToAssets(_strategyStorage(), shares);
```

Comment:
What is the advantage of separating the public and private methods like this? Code paths that have the storage pointer loaded can call the internal/private versions?

Reply:
Yes, _convertToAssets is used by multiple other functions within the contract. So any other function will call the internal one